lib/models/coordinate.dart

```diff
@@ -1,5 +1,7 @@
 import 'dart:collection';
 
```

Comment:
delete newLine and make both imports on the same line

Reply:
done.